tests/suites/test_suite_pkwrite.data

```diff
@@ -1,47 +1,95 @@
 Public key write check RSA
 depends_on:MBEDTLS_RSA_C:MBEDTLS_BASE64_C
```

Comment:
Pre-existing: I think this should rather depend on `MBEDTLS_PEM_PARSE_C` (which in turn depends on `BASE64_C`, making it redundant here).